android/jni/tun2socks/tun2socks.c

```diff
@@ -226,7 +228,8 @@ static void init_arguments (const char* program_name);
 // ==== PSIPHON ====
 
 //==== UPROXY ====
-BAddr dns_server_address;  // IP address of local DNS resolver
```

Comment:
Sounds good. So is it valuable to keep the `//===== PSIPHON ======` comments in the code? They might add noise/distraction/one extra thing to look up for new contributors. If a new contributor shouldn't have to know what Psiphon is to start being productive, might be worth removing at some point (i.e. we can add this to #15). Hope I'm not being too aggressive with my new-contributor hat!